mycity/mycity/utilities/address_utils.py

```diff
@@ -3,18 +3,22 @@
 
 """
 
-from streetaddress import StreetAddressParser
-import mycity.intents.intent_constants as intent_constants
 import logging
 
```

Comment:
should this newline be here since `logging` and `streetaddress` are both library imports?

Reply:
`logging` is stdlib